src/components/Plan.js

```diff
@@ -62,7 +62,9 @@ class Plan extends Component {
                 latitude: station.latitude,
                 longitude: station.longitude,
               }}
-              image = {station.free_bikes == 0 ? require('../../assets/img/bikes_empty.png') : require('../../assets/img/citybike_logo.png')}
+              image = {station.free_bikes == 0 ? require('../../assets/img/bikes_empty.png')
```

Comment:
The nested ternary is not very readable. There is even a [ESLint rule for that](https://eslint.org/docs/rules/no-nested-ternary) 
I would suggest to extract the logic to a seperate function on the same component level and use simply if blocks